### ISSUE TITLE
patroni:system ID mismatch

### ISSUE THREAD

**Issue by ShashiYadav06** [zalando/patroni#2547]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Environment**
-   Patroni version:
-   PostgreSQL version:
-   DCS (and its version):

**Patroni configuration file**
```
Please copy&paste your Patroni configuration file here
```

**patronictl show-config**
```
Please copy&paste the output of "patronictl show-config" command here
```

**Have you checked Patroni logs?**
Please provide a snippet of Patroni log files here

**Have you checked PostgreSQL logs?**
Please provide a snippet here

**Have you tried to use GitHub issue search?**
Maybe there is already a similar issue solved.

**Additional context**

**Comment by ShashiYadav06** [comment]
![image](https://user-images.githubusercontent.com/63842879/215686318-9e56dccd-1778-46a8-8a5f-feade420e5e2.png)
----> here in starting of the image -- getting cluster error but after removing the cluster patroni is getting starting 

![image](https://user-images.githubusercontent.com/63842879/215686584-4e3f711d-63a7-4547-b2b1-a832b9389ca7.png)
-----> here in second image the getting the same error as of first 



Issue which im facing is one starting of one patroni other patroni is giving error of system ID mismatch

note:- have updated the postgres from 11 to 14 using tar file

**Comment by CyberDem0n** [comment]
Rules are the same for everyone, please use issue search before opening a new one

**Comment by ShashiYadav06** [comment]
@CyberDem0n -- have checked then created issue

**Comment by CyberDem0n** [comment]
No, you didn't: https://github.com/zalando/patroni/issues?q=is%3Aissue+system+ID+mismatch+is%3Aclosed
